types/react-final-form-listeners/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Remove all these rules from here and fix errors. new packages cannot disable any lint rules